t/t2350-resource-list.t

```diff
@@ -113,6 +113,59 @@ for input in ${SHARNESS_TEST_SRCDIR}/flux-resource/list/*.json; do
     '
 done
 
+# Special case tests for -q, --queue using fluke.json and fluke.config
+
+FLUKE_CONFIG=${SHARNESS_TEST_SRCDIR}/flux-resource/list/fluke.config
+FLUKE_INPUT=${SHARNESS_TEST_SRCDIR}/flux-resource/list/fluke.json
+test_expect_success 'flux-resource list supports -q, --queue' '
+	flux resource list --queue=debug \
+		--from-stdin --config-file=$FLUKE_CONFIG \
+		< $FLUKE_INPUT > fluke-debug.output &&
+	test_debug "cat fluke-debug.output" &&
+	test_must_fail grep batch fluke-debug.output &&
+	flux resource list --queue=batch \
+		--from-stdin --config-file=$FLUKE_CONFIG \
+		< $FLUKE_INPUT > fluke-batch.output &&
+	test_debug "cat fluke-batch.output" &&
+	test_must_fail grep debug fluke-batch.output &&
+	flux resource list --queue=debug,batch \
+		--from-stdin --config-file=$FLUKE_CONFIG \
+		< $FLUKE_INPUT > fluke-both.output &&
+	grep debug fluke-both.output &&
+	grep batch fluke-both.output
+'
+test_expect_success 'flux-resource list supports -q, --queue' '
+	flux resource list --queue=debug \
+		--from-stdin --config-file=$FLUKE_CONFIG \
+		< $FLUKE_INPUT > fluke-debug.output &&
+	test_debug "cat fluke-debug.output" &&
+	test_must_fail grep batch fluke-debug.output &&
+	flux resource list --queue=batch \
+		--from-stdin --config-file=$FLUKE_CONFIG \
+		< $FLUKE_INPUT > fluke-batch.output &&
+	test_debug "cat fluke-batch.output" &&
+	test_must_fail grep debug fluke-batch.output &&
+	flux resource list --queue=debug,batch \
+		--from-stdin --config-file=$FLUKE_CONFIG \
+		< $FLUKE_INPUT > fluke-both.output &&
+	grep debug fluke-both.output &&
+	grep batch fluke-both.output
+'
```

Comment:
Oh yeah, good catch. I think it is just a duplicate because `flux resource R` uses the same code as `flux resource list`, but might as well add similar tests for `flux resource R` here just using `grep`.

Reply:
Fixed that and force-pushed